src/libraries/System.Diagnostics.EventLog/src/PACKAGE.md

```diff
@@ -0,0 +1,121 @@
+## About
+
+<!-- A description of the package and where one can find more documentation -->
+
+When an error occurs in a Windows machine, the system administrator or support representative must determine what caused the error, attempt to recover any lost data, and prevent the error from recurring. It is helpful if applications, the operating system, and other system services record important events, such as low-memory conditions or excessive attempts to access a disk. The system administrator can then use the Windows Event Log to help determine what conditions caused the error and identify the context in which it occurred.
+
+This package provides the `System.Diagnostics.EventLog.dll` assembly, which contains types that allow applications to interact with the Windows Event Log service.
+
+## Key Features
+
+<!-- The key features of this package -->
+
+* Allows reading from existing logs.
+* Allows writing entries to logs.
+* Can create or delete event sources.
+* Can delete logs.
+* Can respond to log entries.
+* Can create new logs when creating an event source.
+
+## How to Use
+
+<!-- A compelling example on how to use this package with code, as well as any specific guidelines for when to use the package -->
+
+```cs
```

Comment:
@lyndaidaii are code snippets supported like on GitHub?

Reply:
Yes, we support it with syntax highlighting.